admin/view/wp-slimstat-db.php

```
@@ -1516,4 +1516,279 @@ public static function get_your_blog()
+        $cached = intval(wp_slimstat::$wpdb->get_var($sql));
+        return $cached;
+    }
+
+    /**
+     * Get raw goal results as flat array (for Export CSV / Email Reports).
+     * Accepts standard $_args array like get_top().
+     *
+     * @param array $_args Callback args from report definition.
+     * @return array Array of associative arrays with goal_name, uniques, total, cr keys.
+     */
+    public static function get_goals_raw($_args = [])
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$_args'. (undefined)

(UnusedFormalParameter)